config/install/core.entity_form_display.node.localgov_page.default.yml

```diff
@@ -32,13 +32,19 @@ content:
       require_layouts: 1
     third_party_settings: {  }
   localgov_page_summary:
-    type: text_textarea
+    type: string_textarea
```

Comment:
@ekes just checking this is an intentional change.

Reply:
This change was committed to the 1.x branch with https://github.com/localgovdrupal/localgov_page/commit/84ed080af8bfcee098e3d16ea302a414f73cb239, so should disappear with the resolution of the merge conflict.